localstripe/resources.py

```diff
@@ -963,6 +963,25 @@ def _api_list_sources(cls, id, object=None, **kwargs):
 
         return li
 
+    @classmethod
+    def _api_list_all(cls, url, email=None, limit=None, starting_after=None,
+                      **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+        try:
```

Comment:
Can you put an empty line before this one please?

Reply:
Ok